R/missingness.R

```diff
@@ -134,3 +134,60 @@ countMissingData <- function(x, userNAs = NULL) {
   .Deprecated("missingness", "healthcareai")
   missingness(x)
 }
+
+#' Summarizes data given by \code{\link{missingness}}
+#'
+#' @description Interpreting \code{\link{missingness}} results from wide
+#'   datasets is difficult. This function helps interpret missingness output by
+#'   summarizing this output by listing: the percent of variables that
+#'   contain missingness, the variable name of the variable with the maximum
+#'   amount of missingness along with its percent missingness, and a tibble
+#'   that lists the top 5 missingness levels with the count of the number of
+#'   variables associated with each level.
+#' @param object Data frame from \code{\link{missingness}}
+#' @param ... Unused
+#' @return a tibble of the top 5 missingness percentage levels with the count of
+#'   the number of variables associated with each level
+#' @export
+#' @examples
+#' missingness(pima_diabetes) %>%
+#'   summary()
+#'
+summary.missingness <- function(object, ...) {
+  if (!length(object))
```

Comment:
I think you want `if (missing(object))` here

Reply:
@michaellevy That's what I thought originally, but then `summary.model_list` uses this logic. Can you help me understand why `summary.model_list` uses this other logic?

Reply:
That does make sense. Thank you.